tests/src/test/java/alluxio/client/hadoop/FileSystemRenameIntegrationTest.java

```diff
@@ -236,7 +236,7 @@ public void basicRenameTest7() throws Exception {
     // Due to Hadoop 1 support we stick with the deprecated version. If we drop support for it
     // FSDataOutputStream.hflush will be the new one.
     //#ifdef HADOOP1
-    o.sync();
+    o.hsync();
```

Comment:
revert this change since it's a known issue when running unit tests locally.

Reply:
will revert, thanks.